
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:2337369) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_1',
  containerIp: '10.0.3.247',
  mitmPort: 1000,
  autoAccessFixed: 1
}
2022-10-22 20:20:42.958 - [Info] MITM Version: 2
2022-10-22 20:20:42.959 - [Info] Auto Access Enabled: true
2022-10-22 20:20:42.959 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-22 20:20:42.966 - [Info] SSH man-in-the-middle server for 10.0.3.247 listening on 10.0.3.1:1000
2022-10-22 20:23:33.725 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.726 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.726 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.727 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.731 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.732 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.732 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.733 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.733 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.734 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.734 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:33.735 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:35.103 - [Debug] [Connection] Attacker connected: 35.231.251.71 | Client Identification: SSH-2.0-Go
2022-10-22 20:23:51.091 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.091 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.093 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.093 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.093 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.093 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.094 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.094 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.098 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.098 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.117 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.117 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.125 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.125 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.142 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.142 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.144 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.144 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:51.178 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'none'
2022-10-22 20:23:51.178 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-22 20:23:54.943 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:54.944 - [Debug] [Auto Access] Attacker: 35.231.251.71, Threshold: 1, Attempts: 1
2022-10-22 20:23:54.944 - [Debug] [Auto Access] Compromising the honeypot
2022-10-22 20:23:54.944 - [Debug] [Auto Access] Adding the following credentials: 'root:minecraft'
2022-10-22 20:23:55.004 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-22 20:23:55.008 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-22 20:23:55.008 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.247
2022-10-22 20:23:55.012 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.021 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2022-10-22 20:23:55.021 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.022 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.022 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2022-10-22 20:23:55.022 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.023 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.023 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-22 20:23:55.023 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.023 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.024 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2022-10-22 20:23:55.024 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.024 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.029 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-22 20:23:55.030 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.032 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.032 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2022-10-22 20:23:55.033 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.041 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.042 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2022-10-22 20:23:55.042 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.042 - [Debug] [Auth] Attacker 35.231.251.71 trying to authenticate with 'password'
2022-10-22 20:23:55.043 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2022-10-22 20:23:55.043 - [Debug] [Auth] Attacker: 35.231.251.71 has so far made 1 attempts. Remaining: 5 attempts
2022-10-22 20:23:55.071 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-22 20:23:57.627 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.629 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.643 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.644 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.646 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.647 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.648 - [Debug] [Connection] Attacker closed the connection
2022-10-22 20:23:57.648 - [Debug] [Connection] Attacker closed the connection
